src/Mono.Android/Test/Mono.Android-Tests.projitems

```diff
@@ -4,7 +4,9 @@
     <UnitTestApk Include="$(OutputPath)Mono.Android_Tests-Signed.apk">
       <Package>Mono.Android_Tests</Package>
       <InstrumentationType>xamarin.android.runtimetests.TestInstrumentation</InstrumentationType>
-      <ResultsPath>$(MSBuildThisFileDirectory)..\..\..\TestResult-Mono.Android_Tests.xml</ResultsPath>
+      <ResultsPath Condition=" '$(Configuration)' != 'Release' " >$(MSBuildThisFileDirectory)..\..\..\TestResult-Mono.Android_Tests.xml</ResultsPath>
```

Comment:
This is going to get unwieldy fast: the next time we need to generate another set of configurations, this is going to need to be updated again.

Perhaps we should encode things for everything?

```xml
<ResultsPath>$(MSBuildThisFileDirectory)..\..\..\TestResult-Mono.Android_Tests-$(Configuration)+Aot=$(AotAssemblies).xml</ResultsPath>
```